manifest.yml

```diff
@@ -1,30 +1,26 @@
----
-declared-services:
-  retrieve-and-rank-service:
-    label: retrieve_and_rank

```

Comment:
well I got rid of some. I'll do a jshint and more cleanup when there isn't an outstanding PR. Too hard to review/merge if I do it now.